makefiles/simulators/Makefile.modelsim

```diff
@@ -43,7 +43,7 @@ VSIM_ARGS += -onfinish exit
 endif
 
 ifeq ($(GPI_IMPL),vhpi)
-VSIM_ARGS += -foreign \"cocotb_init libfli.so\" -trace_foreign 3
+VSIM_ARGS += -foreign \"cocotb_init libfli.$(LIB_EXT)\" -trace_foreign 3
```

Comment:
Oops, that might explain the terrible performance. Looks like v1_0_1 might
be quite soon.

On 18 February 2015 at 09:02, Chris Higgs <notifications@github.com> wrote:

> In makefiles/simulators/Makefile.modelsim
> <https://github.com/potentialventures/cocotb/pull/214#discussion_r24886927>
> :
>
> > @@ -43,7 +43,7 @@ VSIM_ARGS += -onfinish exit
> >  endif
> >
> >  ifeq ($(GPI_IMPL),vhpi)
> > -VSIM_ARGS += -foreign \"cocotb_init libfli.so\" -trace_foreign 3
> > +VSIM_ARGS += -foreign \"cocotb_init libfli.$(LIB_EXT)\" -trace_foreign 3
>
> Not a comment on this commit, but this has reminded me that we left debug
> on for FLI!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/potentialventures/cocotb/pull/214/files#r24886927>.
>